standard.tex

```diff
@@ -26,67 +26,76 @@
-%end of guiding principles section -- }}}
+If you need to decide between whether to follow a guiding principle or a hard rule, follow the principle. As long as your decisions demonstrably don't violate any of these 4, you're probably ok.
+%-- }}}
 
-%start of hard rules section -- {{{
+%Hard Rules -- {{{
 \section{Rules}
-Here are the rules for all the design and assembly decisions. If you want to break one of these rules, make sure you have a very good excuse. Again though, this document was written by someone who doesn't actually have any idea of what he's doing, so if you think a rule is stupid, feel free to cross it out (provide a reason though). If you think a rule is unclear, feel free to elaborate on it. If you think there's a good rule that should be added, please write it in in the margin or pull request the repository (\url{https://github.com/waterloo-rocketry/waterloo\_rocketry\_electrical\_standard}).
+Here are the rules for all the design and assembly decisions. If you want to break one of these rules, make sure you have a very good excuse. If you think a rule is unclear, feel free to elaborate on it. If you think there's a good rule that should be added, please write it in the margin or pull request the repository (\url{https://github.com/waterloo-rocketry/electrical\_standard}).
 
-%start of connections subsection -- {{{
+%Connections -- {{{
 \subsection{Connections}
 \begin{itemize}
-\item Don't solder stranded wire to protoboard. Don't solder stranded wire to a connector. It gets brittle and it breaks. Use crimped connections, screw terminals, etc instead.
-\item If you're making a crimped connection, use a proper crimp tool if at all possible. Pliers work in a pinch, but the connection is messier and more likely to fail. Good crimp tools are expensive but worth it. (If you're in MME, you can borrow one from MESS.)
-\item If you absolutely need to solder stranded core to something, make sure it's very strain relieved (I personally like P clamps, but zip ties work too).
-\item Crimped connections should be done on stranded wire only. Don't crimp onto solid core. It's weak and it will break, often when you least want it to.
-\item All exposed wire should be heatshrunk; all heatshrink should be clear (IREC regulation).
-\item If you screw up a soldering job (we all do it), redo the job. When soldering surface mount components, inspect solder joints with a small magnifying glass.
+\item Crimped connections are preferable to soldered connections. Bad crimped connections are easier to identify than bad solder joints. Additionally, it is easier to fix a bad crimp in the field since it does not require powered tooling.
+\item If you're making a crimped connection, use a proper crimp tool if at all possible. Pliers are not an acceptable alternative (see ESRA safety critical wiring guidelines). We have proper crimp tools for: generic butt splice and ring/spade connectors, Dupont connectors, and RJ45 connectors.
```

Comment:
we have a ferrule crimper as well

Reply:
I left that out on purpose because our ferrule crimper is awful. It doesn't work at all for small ferrules. Putting it in the standard implies that it can be trusted.